src/snowflakes/registro-table/z-registro-table-expanded-row/index.tsx

```diff
@@ -8,6 +8,7 @@ import { Component, Prop, h } from "@stencil/core";
 export class ZRegistroTableExpandedRow {
   /** Number table column plus 1 for the expand button */
   @Prop() colSpan: number;
+  @Prop() size: "tall" | "medium" | "small" | "none";
```

Comment:
non mi torna che questo modifica sia stata fatto nello sviluppo @leopuleo per caso è qualcosa che avevi visto e rimosso tu in altri sviluppo?